storage/kubernetes/storage.go

```diff
@@ -38,6 +38,8 @@ const (
 type Config struct {
 	InCluster      bool   `json:"inCluster"`
 	KubeConfigFile string `json:"kubeConfigFile"`
+	APIVersion     string `json:"apiVersion"` // API Group and version
```

Comment:
I'd rather not expose this. It's always implied by "useCRD"

Reply:
but how else would users configure the APIVersion?